### ISSUE TITLE
Can Teamtalk add support for zdsr screen readers?

### ISSUE THREAD

**Issue by fltx2004** [BearWare/TeamTalk5#3061]
### Description

Can we increase support for Zhengdu screen reader (ZDSR) in the competition? Implemented through *tolk* calling *zdsrapi*, related links:
[zdsrapi docs](https://www.zdsr.com/docs/api/zdsr-api/)
[zdsrapi sample for c sharp](https://yydjtc.top/download/ZDSRAPI%20sample.zip)

### Application

- [x] qtTeamTalk
- [ ] TeamTalkAndroid
- [ ] iTeamTalk
- [ ] TeamTalkServer

**Comment by bear101** [comment]
TeamTalk uses [Tolk](https://github.com/dkager/tolk) and [Qt's accessibility plugin](https://doc.qt.io/qt-6/accessible.html) on Windows. Your request should be directed there.

**Comment by hwangsihu** [comment]
If we give up on tolk and switch to SRAL, we could enable Orca support along with ZDSR support, but I'm not sure if @bear101 would like that.
I haven't started the implementation yet, but if you agree, we can start the transition.

**Comment by fltx2004** [comment]
You can try

**Comment by hwangsihu** [comment]
I was asking @bear101. I'm sorry, it seems my translator made a mistake.

**Comment by bear101** [comment]
> If we give up on tolk and switch to SRAL, we could enable Orca support along with ZDSR support, but I'm not sure if [@bear101](https://github.com/bear101) would like that. I haven't started the implementation yet, but if you agree, we can start the transition.

I'm fine with giving SRAL a try. Tolk is not being developed anymore so SRAL sounds like a better option.

**Comment by BlindMaster24** [comment]
> > If we give up on tolk and switch to SRAL, we could enable Orca support along with ZDSR support, but I'm not sure if [@bear101](https://github.com/bear101) would like that. I haven't started the implementation yet, but if you agree, we can start the transition.
> 
> I'm fine with giving SRAL a try. Tolk is not being developed anymore so SRAL sounds like a better option.

https://github.com/ethindp/prism

**Comment by hwangsihu** [comment]
Ah, I forgot that NVGT switched to that. That seems like a better choice.

**Comment by fltx2004** [comment]
Perhaps there is a modified [tolk](/boz700908/tolk )? Maybe you can use this?

**Comment by hwangsihu** [comment]
I will just use Prism. It is cross-platform and supports many screen readers. Since it also supports UIA, it can support all screen readers that support UIA.
